### PR TITLE
[Docs] PR for Missing Information about add node

### PR DESCRIPTION
**Pull request by lizayugabyte**

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ The action to release a node instance is available from the Removed internal sta
 Taking this action transfers the node to a BeingDecommissioned and then Decommissioned internal state, as follows:
 
 1. There is a wait for Master leader.
-2. This server is removed from blacklist on Master leader (blacklisting is described in [Remove node](#remove-node)).
+2. This server is removed from blacklist on Master leader (blacklisting is described in [Remove node](#remove-node/)).
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md:69]: Link broken. Pretty sure you don't add trailing slash on in-page links

File: docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md [hunk 2]
```diff
@@ -87,3 +87,5 @@ Taking this action completely eliminates the node, as follows:
 
 1. Removes the node record from the universe metadata.
 2. Updates metadata in the database only.
+
+You can recover a deleted node by following instructions provided in [Use Yugabyte Platform to add a node](../add-nodes/).
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md:91]: Reading this page, not clear to me that you can recover a 'deleted' node; in Add Node it says you can add back a node in 'decommissioned' status, which comes after Release Node Instance steps. Deleted node may be gone for good? Needs clarification.

**Review by ddhodge — APPROVED** [review]
2 comments

**Comment by ddhodge** [comment]
The way these topics read, it appears you can only add a node back, after removing it

1. should Add a Node be moved after Eliminate in the left nav? 
2. should Add a Node be renamed Revive a node? 
3. should the Revive node page should be incorporated into Eliminate if it can only be done in that context?

Can you spontaneously add a brand new node using this process? If this is only about reviving nodes, then I would do 3.

File: docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md:69]: fixed

File: docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md:91]: Good catch. I moved that sentence to under "Release node instance".
(this whole article was really carefully crafted based on the input from dev, as internal node states and displayed statuses weren't always a match)

**Pull request merged**